### ISSUE TITLE
Missing parentheses and quotation marks on sequence

### ISSUE THREAD

**Issue by mjkl-gh** [sequelize/sequelize-auto#446]
Using postgres 10 I have a primary key called pk_measure_update. When generating the models from my database the following is loaded:

```
    pk_measure_update: {
      type: DataTypes.BIGINT,
      allowNull: false,
      defaultValue: nextvalseq_measure_update::regclass,
      primaryKey: true
    },
```

Somehow, the parentheses and quotation marks are missing for the sequence. I expected:

```
    pk_measure_update: {
      type: DataTypes.BIGINT,
      allowNull: false,
      defaultValue: "nextval(seq_measure_update::regclass)",
      primaryKey: true
    },
```

Currently on a bit of a deadline. I do want to supply more context at a later moment. If anyone can point me in the direction they want context about in the meantime, I would be really grateful.

**Comment by steveschmitt** [comment]
Thanks, I'll look into it.

**Comment by mjkl-gh** [comment]
On second note  defaultValue: "nextval(seq_measure_update::regclass)", also didn't work although I just might remember that was due to permissions on the sequence. Will get back on that later

Changed it to autoIncrement: true and it worked. This translates to the value DEFAULT for postgres according to the logs.

**Comment by steveschmitt** [comment]
So what do you think should be the correct defaultValue in the Sequelize model?

**Comment by steveschmitt** [comment]
Fixed postgres autoIncrement in commit 62bbe35a

**Comment by steveschmitt** [comment]
Postgres `autoIncrement` is fixed in 0.7.3
